app/routes.py

```diff
@@ -0,0 +1,103 @@
+from flask import Blueprint, json, jsonify, request, make_response
+from app.models.customer import Customer
+from app.models.video import Video
+from app import db
+from flask_sqlalchemy import model
+from sqlalchemy import func
+import requests
+from datetime import timedelta
+
+from tests.test_wave_02 import CUSTOMER_NAME
+
+
+videos_bp = Blueprint("videos", __name__, url_prefix="/videos")
+
+@videos_bp.route("", methods=["GET", "POST"])
+def handle_videos():
+    if request.method == "GET":
+        videos = Video.query.all()
+        videos_response = []
+        for video in videos:
+            videos_response.append(video.create_video_dict())
+        return jsonify(videos_response)
+
+    if request.method == "POST":
+        request_body = request.get_json()
+        if "title" not in request_body:
+            return make_response ({"details": "Request body must include title."}), 400
+        elif "release_date" not in request_body: 
+            return make_response({"details":"Request body must include release_date."}, 400)
+        elif "total_inventory" not in request_body:
+            return make_response({"details":"Request body must include total_inventory."}, 400)
+        
+        new_video = Video(
+            title = request_body["title"],
+            release_date = request_body["release_date"],
+            total_inventory = request_body["total_inventory"]
+            )
+        db.session.add(new_video)
+        db.session.commit()
+            
+        return jsonify(new_video.create_video_dict()), 201
+
+@videos_bp.route("/<video_id>", methods=["GET", "DELETE", "PUT"])
+def handle_video_by_id(video_id):
+    try:
+        int(video_id)
+    except:
+        return_message = {"message": f"Video {video_id} was not found"}
+        return make_response(return_message, 400)
+
+    video= Video.query.get(video_id)
+    if video == None:
+        return_message = {"message": f"Video {video_id} was not found"}
+        return make_response(return_message, 404)
```

Comment:
It's more _pythonic_ to use `is` when comparing values to `None` according to PEP8. 

> Comparisons to singletons like None should always be done with is or is not, never the equality operators.

https://www.python.org/dev/peps/pep-0008/#programming-recommendations

```suggestion
    if video is None:
        return_message = {"message": f"Video {video_id} was not found"}
        return make_response(return_message, 404)
```